packages/upload-client/src/tools/isFileData.ts

```diff
@@ -1,4 +1,7 @@
-import { SupportedFileInput, ReactNativeAsset } from '../types'
+import { ReactNativeAsset, SupportedFileInput } from '../types'
+import { isBuffer } from './isBuffer.node'
```

Comment:
It will be replaced with corresponding file (node, browser, react-ntive) in the build-time